cmd/snap-exec/main_test.go

```diff
@@ -215,7 +215,7 @@ func (s *snapExecSuite) TestSnapExecAppRealIntegration(c *C) {
 	})
 
 	canaryFile := filepath.Join(c.MkDir(), "canary.txt")
-	script := filepath.Join(dirs.GlobalRootDir, "/snap/snapname/42/run-app")
+	script := fmt.Sprintf("%s/snapname/42/run-app", dirs.SnapMountDir)
```

Comment:
I know we're not looking for portability to places where `/` isn't the dir separator, but any reason for the switch to `fmt.Sprintf` instead of `filepath.Join`?